raiden/api/rest.py

```diff
@@ -95,7 +95,7 @@
 
 def api_response(result, status_code=HTTPStatus.OK):
     response = make_response((
-        json.dumps(result),
```

Comment:
This doesnt really fix the problem, if this function is called with `status_code=HTTPStatus.NO_CONTENT` the error will persist.

Reply:
No the error does not persist. I tested it myself. Having a response message of `''` is length 0 and as such pywsgi is happy with it.

Reply:
try this:

`api_response('a', status_code=HTTPStatus.NO_CONTENT)`

Reply:
I meant it solves the problem reported by the users.

Yes if you want the function to enforce no content we need our own assertion there, but that is basically a programmers error. I can put it.